app/assets/javascripts/models/page/search_page.js.coffee

```diff
@@ -50,7 +50,7 @@ ns.SearchPage = do (ko
     $('.launch-customize-modal').click ->
       $('#customizeDataModal').modal('show')
 
-    setTimeout initModal, 2000 if !window.edscportal
+    preferences.ready.done(-> initModal()) if !window.edscportal
 
   class SearchPage
     constructor: ->
```

Comment:
The "evidence" of multiple initializations can be seen in the js console. Everytime the preferences is initialized, an ajax call is made which is logged in the console.

The two consecutive calls to preferences will possibly make the code read dirty data if the preferences are changed between the two calls.